models/aeif_cond_beta_multisynapse.h

```diff
@@ -51,20 +51,66 @@
 
  Description:
 
- aeif_cond_beta_multisynapse is an extension of
- aeif_cond_alpha_multisynapse.  It allows an arbitrary number of synaptic
- rise time and decay time constant. Synaptic conductance is modeled by a
+ aeif_cond_beta_multisynapse is a conductance based adaptive exponential
+ integrate-and-fire neuron model. It allows an arbitrary number of synaptic
+ rise time and decay time constants. Synaptic conductance is modeled by a
  beta function, as described by A. Roth and M.C.W. van Rossum
  in Computational Modeling Methods for Neuroscientists, MIT Press 2013,
  Chapter 6
 
- The time constants are supplied by two arrays taus_rise and taus_decay for
- the synaptic rise time and decay time, respectively.  Port numbers
- are then automatically assigned and there range is from 1 to n.  (n
- being the index of the last element of the tau_ex and tau_in
- arrays).
+ The number of receptor ports is supplied by the function "n_receptors".
+ The time constants are supplied by two arrays, "taus_rise" and "taus_decay" for
+ the synaptic rise time and decay time, respectively. The synaptic
+ reversal potentials are supplied by the array "E_rev". The port numbers
+ are automatically assigned in the range from 1 to n_receptors.
  During connection, the ports are selected with the property "receptor_type".
 
+ The membrane potential is given by the following differential equation:
+ C dV/dt = -g_L(V-E_L) + g_L*Delta_T*exp((V-V_T)/Delta_T) + I_syn_tot(V, t)
+           - w + I_e
+
+ where:
+ I_syn_tot(V,t) = \sum_i g_i(t) (V - E_{rev,i})
+
+ and the synapse i is excitatory or inhibitory depending on the value of
+ E_{rev,i}.
+
```

Comment:
The differential equation for w fell out in one of your previous commits. I think it should be here. Would you add it again?